plugins/meta-inject/impl/src/main/java/org/pentaho/di/trans/steps/metainject/MetaInjectMeta.java

```diff
@@ -70,9 +70,12 @@
 import java.util.Map;
 import java.util.Map.Entry;
 
```

Comment:
@bconlon-hv Please update the copyright date to reflect 2023
Also, welcome to Pentaho!